2017/Lab06/src/Methods.java

```diff
@@ -0,0 +1,119 @@
+import java.util.ArrayList;
+
+/**
+ * @author king
+ *
+ */
+public class Methods {
+	
+	char letter = 'e';
+	/**
+	 * counts the number of occurrences of the character "e" in a string.
+	 * @param s
+	 * @param letter
+	 * @return
+	 */
+	public static int occurunces(String s, char letter){
+		if(s.length() == 0){
+			return 0;
+		}
+		int count = 0;
+		if(s.charAt(0) == letter){
+				count++;
+		}
+		return count + occurunces(s.substring(1), letter);
```

Comment:
When it comes to this line, count can either be 0 or 1, right? Therefore, wouldn't be better to write this with if and else rather than having count?